CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributing to Rand
+
+Thank you for your interest in contributing to Rand!
+
+The following is a list of notes and tips for when you want to contribute to
+Rand with a PR.
+
+If you want to make major changes, it is usually best to open an issue to
+discuss the idea first.
+
+Rand doesn't (yet) use rustfmt. It is best to follow the style of the
+surrounding code, and try to keep an 80 character line limit.
+
+
+## Documentation
+
+We especially welcome documentation PRs.
+
+As of Rust 1.25 there are differences in how stable and nightly render
+documentation links. Make sure it works on stable, then nightly should be good
```

Comment:
Another handy tip is to install inotify-tools, then use this command to rebuild documentation automatically on save:

```
while inotifywait -r -e close_write src/; do cargo doc; done
```

Reply:
That trick is new for me. It is nice, but it does not really seem suitable here in my opinion (and Linux-specific?).

Reply:
Yes, linux-specific. Better:

while inotifywait -r -e close_write src/ rand_core/; do cargo doc; done